### ISSUE TITLE
TextField needed

### ISSUE THREAD

**Issue by CodingItWrong** [bakerac4/glimmer-native#10]
Am I correct in seeing that TextField has not yet been implemented? I don't see it in the list in the readme, and if I try to use `<TextField />` I get the error:

```
*** Terminating app due to uncaught exception 'NativeScript encountered a fatal error: TypeError: Right side of assignment cannot be destructured
at
1   lookupComponent@file:///app/vendor.js:29891:20
```

**Comment by CodingItWrong** [comment]
I see that `TextView` is available, though, so that's cool to get started.

The `text` attribute on `TextView` doesn't seem to be two-way bound: it allows me to provide the initial text shown, but when I edit it, that doesn't make its way back to the bound and tracked attribute. Is there a different way I should be using to get access to the updated text?

**Comment by CodingItWrong** [comment]
For others' future reference, looks like an explicit event handler works:

```
    <TextView
      text={{this.newTodo}}
      {{on "textChange" (action handleChangeText)}}
    />
```

```
    handleChangeText(event) {
      this.newTodo = event.value
    }
```

**Comment by bakerac4** [comment]
Closed by #12
